spec/suites/layer/GeoJSONSpec.js

```diff
@@ -512,3 +512,387 @@ describe("L.LayerGroup#toGeoJSON", function () {
 		});
 	});
 });
+
+describe("L.GeoJSON functions", function () {
+	describe("#geometryToLayer", function () {
+		const point = {
+			type: "Point",
+			coordinates: [0, 0]
+		};
+		const multiPoint  = {
+			type: "MultiPoint",
+			coordinates: [
+				[0, 0], [10, 10]
+			]
+		};
+		const line =  {
+			type: "LineString",
+			coordinates: [
+				[0, 0], [10, 10], [20, 20]
+			]
+		};
+		const multiLine = {
+			type: "MultiLineString",
+			coordinates: [
+				[[10, 10], [20, 20], [30, 30]],
+				[[50, 50], [60, 60], [70, 70]]
+			]
+		};
+		const polygon = {
+			type: "Polygon",
+			coordinates: [
+				[[30, 10], [40, 40], [20, 40], [10, 20], [30, 10]]
+			]
+		};
+		const multiPolygon = {
+			type: "MultiPolygon",
+			coordinates: [
+				[
+					[[30, 20], [45, 40], [10, 40], [30, 20]]
+				],
+				[
+					[[15, 5], [40, 10], [10, 20], [5, 10], [15, 5]]
+				]
+			]
+		};
+		const geometryCollection  = {
+			type: "GeometryCollection",
+			geometries: [
+				{
+					type: "Point",
+					coordinates: [0, 0]
+				},
+				{
+					type: "LineString",
+					coordinates: [
+						[10, 10], [20, 20]
+					]
+				}
+			]
+		};
+
+		function customPointToLayer(geojsonPoint, latLng) {
+			return new L.Circle(latLng, {
+				radius: geojsonPoint.properties.radius
+			});
+		}
+
+		function customCoordstoLatLng(coords) {
+			return new L.LatLng(coords[1] + 1, coords[0] + 1, coords[2] + 1);
```

Comment:
But in other part you prefer factory form (`L.latLng()`).
You should be consistent everywhere - either prefer constructors (with `new`), either factories.

Reply:
Yeah true! Just saw that too late and then it was pushed. I wanted to wait for a review first...